data/transaction.go

```diff
@@ -34,7 +32,7 @@ type Transaction struct {
 	SenderUserName       []byte        `json:"senderUsername,omitempty"`
 	ReceiverUserName     []byte        `json:"receiverUsername,omitempty"`
 	Log                  TxLog         `json:"-"`
-	rcvAddrBytes         []byte
+	RcvAddrBytes         []byte        `json:"-"`
```

Comment:
ReceiverAddressBytes ? (+ rename the according getter?)

Reply:
done.